internal/security/credentials.go

```diff
@@ -15,10 +15,16 @@
 package security
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/edgexfoundry/app-functions-sdk-go/internal/common"
 	"github.com/edgexfoundry/app-functions-sdk-go/internal/store/db"
 )
 
+// GetDatabaseCredentials retrieves the login credentials for the database
+// If security is disabled or the database is Redis then we use the credentials supplied by the
+// configuration.
```

Comment:
Need to change this to get the DB creds from InsecureSecrets rather than the passed in DatabaseInfo when in non-secure mode.